src/gui/modals/SetupDialog.cpp

```diff
@@ -1240,6 +1252,22 @@ void SetupDialog::resetBufferSize()
 	setBufferSize(DEFAULT_BUFFER_SIZE / BUFFERSIZE_RESOLUTION);
 }
 
+void SetupDialog::updateSampleRates()
+{
+	m_sampleRate->clear();
+	if (m_audioInterfaces->currentText() == "JACK (JACK Audio Connection Kit)")
```

Comment:
The strings for audio interface name could be changed due to translations and future code changes. According to line 493 and 536-549, the value will be `AudioDeviceSetupWidget::tr(AudioJack::name())`.
I think you should update the combo box based on the `AudioDeviceSetupWidget` child class to be selected.

Reply:
Good point, but as side question, do we really have a good reason to localise the backend names? They are their Names after all.